quic/s2n-quic-core/certs/config/ca.cnf

```diff
@@ -0,0 +1,21 @@
+[ req ]
+prompt = no
+distinguished_name = ca_distinguished_name
+x509_extensions=ext
+
+[ ca_distinguished_name ]
+# country name
+C = JP
```

Comment:
Why these values and not just Seattle?

Reply:
I borrowed from existing cnf file in the project: https://github.com/aws/s2n-quic/blob/main/examples/s2n-mtls/certs/config/ca.cnf

Reply:
Ah. I guess that's fine?